webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
remember that you will have to improve it before build for productiion ;)

Reply:
hm? :)

Reply:
your webpack configuration is not ready to build production version of the App. But it is not for this PR, add to todos.